frontend/resourceadm/components/ImportResourceModal/ImportResourceModal.tsx

```diff
@@ -187,8 +97,15 @@ export const ImportResourceModal = ({
           label={t('resourceadm.dashboard_import_modal_select_env')}
         />
       </div>
-      {displayService()}
-      {displayTitleAndId()}
+      {selectedEnv && (
+        <ServiceContent
+          selectedContext={selectedContext}
+          env={selectedEnv}
+          selectedService={selectedService}
+          onSelectService={(a2ls: Altinn2LinkService) => setSelectedService(a2ls)}
```

Comment:
In my personal opinion, I tend to prefer longer, more descriptive variable names. 🙈 However, please understand that this is solely a matter of personal preference, and I have no intention of imposing my choice on you. Nevertheless, if the variable is utilized frequently throughout the codebase, using shorter names could potentially make the code more challenging to comprehend, particularly if there's a prevalence of concise names scattered throughout the code.

```suggestion
          onSelectService={(altinn2LinkService: Altinn2LinkService) => setSelectedService(altinn2LinkService)}
```

Reply:
Yes, I agree 😄